README.md

```diff
@@ -112,99 +172,140 @@ The job will appear on the WebUI and you can inspect/restart/kill it.
+or even inside the toplevel), see the [documentation of the EDSL API](src/lib/eDSL.mli).
 
 
 ### Example
 
-This example is a “single-target” workflow that runs an arbitrary shell command
-on an [LSF-based](http://en.wikipedia.org/wiki/Platform_LSF) cluster:
+The following script extends the previous shell based example with the capability to send emails upon the success or failure of your command.
 
 ```ocaml
 #use "topfind"
 #thread
 #require "ketrew"
-let run_command_with_lsf cmd =
+
+let run_command_with_daemonize ~cmd ~email =
   let module KEDSL = Ketrew.EDSL in
-  let host =
-    (* `Host.parse` takes an URI and creates a “Host” datastructue: a place to
-       run stuff.  *)
-    KEDSL.Host.parse
-      "ssh://user42@MyLSFCluster/home/user42/ketrew-playground/?shell=bash"
-    (* This one is an SSH host, named `MyLSFCluster`.
-       The directory `/home/user42/ketrew-playground/` will be used by Ketrew
-       to monitor the jobs. *)
-  in
-  let program =
-    (* A “program” is a datastructure representing “extended shell scripts”.
-       `Program.sh` creates one out a shell command. *)
-    KEDSL.Program.sh cmd in
-  let lsf_build_process =
-    (* “build process” is a method for making things:
-       `lsf` creates a datastructure that represents a job running a `program`
-       with the LSF scheduling engine, on the host `host`.  *)
-    KEDSL.lsf
-      ~queue:"normal-people" ~wall_limit:"1:30"
-      ~processors:(`Min_max (1,1)) ~host program
+
+  (* Where to run stuff *)
+  let host = KEDSL.Host.tmp_on_localhost in
+
+  (* A “program” is a datastructure representing an “extended shell script”. *)
+  let program = KEDSL.Program.sh cmd in
+
+  (* A “build process” is a method for making things.
+     
+     In this case, `daemonize` creates a datastructure that represents a job
+     running our program on the host. *)
+  let build_process = KEDSL.daemonize ~host program in
+
+  (* A target that Ketrew will activate after the success of cmd *)
+  let email_target ~success =
+    let sstring = if success then "succeeded" else "failed" in
+    let e_program =
+      KEDSL.Program.shf "echo \"'%s' %s\" | mail -s \"Status update\" %s"
+        cmd sstring
+        email
+    in
+    let e_process =
+      KEDSL.daemonize ~using:`Python_daemon ~host e_program in
+    KEDSL.target ("email result " ^ sstring) ~make:e_process 
   in
-  (* The function `KEDSL.target` creates a node in the workflow graph.
-     This one is very simple, it has a name and a build-process,
-     and since it doesn't have dependencies or fallbacks, it is a
-     “single-node” workflow: *)
-  KEDSL.target
-     "run_command_with_lsf"
-     ~make:lsf_build_process
+
+  (* The function `KEDSL.target` creates a node in the workflow graph. *)
+  KEDSL.target "daemonize command"
+    ~make:build_process
+    ~on_success_activate:[email_target true]
+    ~on_failure_activate:[email_target false]
```

Comment:
I find a bit weird to introduce `on_*_activate` before dependencies (which are the normal way of doing things).